src/invidious/helpers/logger.cr

```diff
@@ -10,7 +12,7 @@ enum LogLevel
 end
 
 class Invidious::LogHandler < Kemal::BaseLogHandler
-  def initialize(@io : IO = STDOUT, @level = LogLevel::Debug)
+  def initialize(@io : IO = STDOUT, @level = LogLevel::Debug, @color : Bool = true)
```

Comment:
Pretty minor nitpick but I'll prefer if you can change `@color` to something that doesn't collide with `#color`. Such as `@use_color`.

It is valid Crystal code but it might cause some confusion to someone starting out with Crystal, or when switching from another language.